pkg/codegen/operations.go

```diff
@@ -111,6 +111,18 @@ func (p ParameterDefinitions) FindByName(name string) *ParameterDefinition {
 	return nil
 }
 
+// GoName converts parameter names to CamelCase and for
+func GoName(name string) string {
+	if name == "id" {
+		return "ID"
+	}
+
+	name = ToCamelCase(name)
+	name = strings.Replace(name, "Url", "URL", -1)
```

Comment:
I don't know if I like this....

Go has many of these capitalizations, why handle only two? We can perhaps do the whole list, but my preference would be to simply not do it at all - and pass through the names in the oapi schema as best we can.